vars/commonPipelineEnvironment.groovy

```diff
@@ -1,3 +1,5 @@
+import groovy.transform.Field
```

Comment:
Is this unused import needed for the test?

Reply:
no, was a stupid left over, removed ...